packages/strapi-plugin-content-manager/admin/src/components/InputUID/RightLabel.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Success, Remove } from '@buffetjs/icons';
+import styled from 'styled-components';
+import { useGlobalContext } from 'strapi-helper-plugin';
+
+import pluginId from '../../pluginId';
+
+// This component is only used in this file so there is no need to create a separated file.
+const RightContentLabel = styled.div`
+  padding: 0 5px;
+  text-transform: capitalize;
+  color: ${props => props.theme.main.colors[props.color]};
+`;
+
+const RightLabel = ({ label, availability }) => {
+  const { formatMessage } = useGlobalContext();
+
+  if (label) {
+    return (
+      <RightContentLabel color="blue">
+        {formatMessage({
+          id: `${pluginId}.components.uid.regenerate`,
+        })}
+      </RightContentLabel>
+    );
+  }
+
+  if (availability !== null) {
```

Comment:
I don't understand when availability can be null? At the initialisation?

Reply:
At the initialization and if the UID is available, it will be null after 4 seconds.

Reply:
ok